.buildkite/scripts/steps/dra-publish.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -uo pipefail
+
+source .buildkite/scripts/bootstrap.sh
+
+echo "+++ Setting DRA params"
+# Shared secret path containing the dra creds for project teams
+DRA_CREDS=$(vault kv get -field=data -format=json kv/ci-shared/release/dra-role)
```

Comment:
we normally collect the required credentials inside the pre-command hook, similar to the docker creds.